save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/DemoStatus.kt

```diff
@@ -4,37 +4,38 @@ import kotlinx.serialization.Serializable
 
 /**
  * Enum that defines demo status
+ * @property statusTooltip
  */
 @Serializable
-enum class DemoStatus {
+enum class DemoStatus(val statusTooltip: String) {
     /**
      * Demo is created but stopped by error
      */
-    ERROR,
+    ERROR("Oops, some issue has occurred. Please, check logs and contact us."),
 
     /**
      * No demo created yet
      */
-    NOT_CREATED,
+    NOT_CREATED("Your demo is not created yet. Time to fix it!"),
 
     /**
      * Demo is ready for use
      */
-    RUNNING,
+    RUNNING("Demo is successfully started and ready to use!"),
 
     /**
      * Demo is already created but not ready for use yet
      */
-    STARTING,
+    STARTING("We need some time to start your demo. Give us a moment and it's gonna be ready!"),
 
     /**
      * Demo is created but stopped by owner
      */
-    STOPPED,
+    STOPPED("Demo is created, but not started yet. You can reconfigure it and start."),
```

Comment:
`was stopped` or `finished`? or it's really `not yet started`?

Reply:
Fixed